json/src/main/java/io/airlift/json/JsonCodec.java

```diff
@@ -215,6 +215,11 @@ public byte[] toJsonBytes(T instance)
         }
     }
 
+    public ObjectMapper mapper()
+    {
+        return mapper;
```

Comment:
Mapper is mutable (and even potentially shared `io.airlift.json.JsonCodec#OBJECT_MAPPER_SUPPLIER`). This breaks encapsulation. This is a no-go from my perspective.

Reply:
I'll find another solution.

Reply:
@findepi I've come up with an alternate. PTAL.

Reply:
also `return mapper.copy();` would do, right?